tiledb/sm/misc/constants.cc

```diff
@@ -726,6 +726,21 @@ const void* fill_value(Datatype type) {
   return nullptr;
 }
 
+#ifdef __linux__
+/** Possible certificate files; stop after finding one.
+ * inspired by https://golang.org/src/crypto/x509/root_linux.go
```

Comment:
link to hash: https://github.com/golang/go/blob/f0e940ebc985661f54d31c8d9ba31a553b87041b/src/crypto/x509/root_linux.go

Reply:
Thanks updated